### PR TITLE
CBO-101: LGFS Grad fees design implementation

### PR DESCRIPTION
**Pull request by cleargif**
#### What
Updated layout for the Grad fees page

#### Ticket

https://dsdmoj.atlassian.net/browse/CBO-101

#### Why
Simon asked nicely

#### How
- Moved page onto the v2 CSS 
- Rebuilt the page taking out the `grid-row` layout
- Updated translations
- Extended cucumbers to include all fields on the page
- Updated error state CSS
- Manual error deep links :(

### CODE REVIEW

**Comment by cleargif** [comment]
@colinbruce @jsugarman 

What's up with this `diff-coverage` and `total-coverage`

Not seen this before..

File: config/locales/en.yml [hunk 1]
```diff
@@ -147,7 +147,7 @@ en:
         supplier_number:
           attributes:
             supplier_number:
-#              invalid: *invalid_supplier_number
+              #invalid: *invalid_supplier_number
```
Comment (lostie) [inline config/locales/en.yml]: Can this just be removed if it's not in use?

File: features/step_definitions/litigator_claim_steps.rb [hunk 2]
```diff
@@ -89,4 +97,3 @@
 def additional_supplier_numbers
   %w[1A833H 1A832G 1A831F 1A830E 1A829D 1A828C 1A827B 1A826A 1A825Z 1A824Y 1A823X 1A822W 1A821V 1A820U]
 end
-
```
Comment (lostie) [inline features/step_definitions/litigator_claim_steps.rb:92]: 🏆

**Review by lostie — APPROVED** [review]
LGTM 👍

**Pull request merged**